.github/workflows/ci.yml

```diff
@@ -3,7 +3,10 @@
 
 name: CI
 
-on: [push]
+on:
```

Comment:
Note that the build would not run for every push, @butenkor @praveenkumarct is this what you wanted?

Reply:
I am still working on this, Build was not working for the schedule. So I tried this. 
Will ask for the review, Once i am done with the changes.

Reply:
Correct. Just discussed in in slack with @praveenkumarct